Common/include/option_structure.hpp

```diff
@@ -1681,7 +1681,7 @@ static const MapType<std::string, OUTPUT_TYPE> Output_Map = {
   MakePair("CGNS", OUTPUT_TYPE::CGNS)
   MakePair("SURFACE_CGNS", OUTPUT_TYPE::SURFACE_CGNS)
   MakePair("STL_ASCII", OUTPUT_TYPE::STL_ASCII)
-  MakePair("STL_BINARY", OUTPUT_TYPE::STL_BINARY)
+  MakePair("STL", OUTPUT_TYPE::STL_BINARY)
```

Comment:
This is not what we talked about. No "silent changes" please, if someone had STL in their config, this changes the behaviour silently from ASCII to binary.
```suggestion
  MakePair("STL_BINARY", OUTPUT_TYPE::STL_BINARY)
```